src/scss/components/people/_people

```diff
@@ -0,0 +1,25 @@
+@charset "UTF-8";
+
+.su-person-content {
+    
+    // right sidebar
+    .right-region {
+
+        // heading
+        .block__title {
+            font-size: 2rem !important;
```

Comment:
You could do soemthing like this to not use the `important`. 
`.content .su-person-content .right-region h2.block__title`

Reply:
Because the original css is very specific, to override this we would have to call out each h2 in that sidebar. To fix it I used one of the wrapper ID's to encompass all the css and removed the important.